compute_tools/src/migrations/README.md

```diff
@@ -0,0 +1,3 @@
+# `compute_ctl` Migrations
+
+These migrartions run at TODO.
```

Comment:
Just a reminder here to make sure you fill this in :)

Reply:
That file should've been deleted. Whoops